.env.development

```diff
@@ -0,0 +1 @@
+REACT_APP_HOMIE_API=http://localhost:4000/homies
```

Comment:
This shouldn't be required if you configured a proxy. 
```suggestion
```

Reply:
Thank you 👍🏼I deleted and saw that I hadn't used environment variable anywhere 🙈😅